Sources/CreateOptions/PackageDeclaration.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+
+public enum DependencyRule: Decodable {
+    case exact(version: String)
+    case upToNextMajor(from: String)
+    case upToNextMinor(from: String)
+    case range(from: String, to: String)
+    case closedRange(from: String, to: String)
+    case branch(name: String)
+    case commit(hash: String)
+    case from(version: String)
+
+    var declaration: String {
+        switch self {
+        case .exact(version: let version):
+            return ".exact(\"\(version)\")"
+        case .upToNextMajor(from: let version):
+            return ".upToNextMajor(from: \"\(version)\")"
+        case .upToNextMinor(from: let version):
+            return ".upToNextMinor(from: \"\(version)\")"
+        case .range(from: let min, to: let max):
+            return "\"\(min)\"..<\"\(max)\""
+        case .closedRange(from: let min, to: let max):
+            return "\"\(min)\"...\"\(max)\""
+        case .branch(name: let name):
+            return ".branch(\"\(name)\")"
+        case .commit(hash: let hash):
+            return ".revision(\"\(hash)\")"
+        case .from(let version):
+            return "from: \"\(version)\""
+        }
+    }
```

Comment:
For reference form my last comment, when we change the package-tools-version to 5.6, this will become the following:

 ```suggestion
    var declaration: String {
        switch self {
        case .exact(version: let version):
            return "exact: \"\(version)\""
        case .range(from: let min, to: let max):
            return "\"\(min)\"..<\"\(max)\""
        case .closedRange(from: let min, to: let max):
            return "\"\(min)\"...\"\(max)\""
        case .branch(name: let name):
            return "branch: "\(name)\""
        case .commit(hash: let hash):
            return "revision: "\(hash)\""
        case .from(let version):
            return "from: \"\(version)\""
        }
    }
```